modules/ingestor/src/graph/sbom/spdx.rs

```diff
@@ -72,6 +72,25 @@ fn suppliers(sbom: &SPDX) -> Vec<String> {
     Vec::from_iter(result)
 }
 
+/// Extract versions for a SPDX SBOM by collecting versions of describing packages
+fn versions(sbom: &SPDX) -> Vec<String> {
+    // packages describing the SBOM
+    let describing = describing_packages(sbom);
```

Comment:
Initially I'd prefer to have packages (or their versions) returned right away. Taking a look at the implementation of `describing_packages` that's however not really possible, as one source of this information only holds the string, not the full package information.

I think the compromise (searching through everything once again) it ok, but we should document this here.